lib/benchee/statistics.ex

```diff
@@ -322,11 +322,16 @@ defmodule Benchee.Statistics do
     %Suite{suite | scenarios: new_scenarios}
   end
 
-  defp iterations_per_second(average_microseconds) do
+  defp iterations_per_second(0.0, _), do: 0.0
+
+  defp iterations_per_second(_, :memory), do: 0.0
```

Comment:
mentioned before, but I think `nil` might be a better value here. That leads to us having to handle it in output but I believe nil is the more correct value here.